packages/website/src/pages/feedback/ColorDot.tsx

```diff
@@ -1,13 +1,11 @@
 import * as React from 'react';
-import {Section} from '@coveord/plasma-react';
+import {PageLayout} from '../../building-blocs/PageLayout';
 
-import PlasmaComponent from '../../building-blocs/PlasmaComponent';
+const code = `

```

Comment:
I think I would put this one as the main example 
![image](https://user-images.githubusercontent.com/35579930/158193693-d81eb845-fbfc-4173-96ad-b6ebe27645b9.png)